apps/explorer/src/utils/stringUtils.ts

```diff
@@ -57,6 +57,11 @@ export function formatAddress(address: string) {
     )}`;
 }
 
+export function formatDigest(digest: string) {
+    // Use 6 characters prefix to match the address format
```

Comment:
cc @mystie711 what is the expected truncation for transaction digests?

Reply:
Good one. Let's do 10 characters ending with compact ellipsis.
It will offer good visual contrast to other items in the column.
So, "12E45G78p0..." 

@Jibz1

Reply:
Done